modules/apps/data-engine/data-engine-taglib/src/main/java/com/liferay/data/engine/taglib/servlet/taglib/DataLayoutBuilderTag.java

```diff
@@ -146,6 +146,18 @@ private String _getDefaultLanguageId() {
 		return ddmStructure.getDefaultLanguageId();
 	}
 
+	private String _getFieldsPluginEntryPoint() {
```

Comment:
The diff of this file is very confusing 🤷‍♂️

Reply:
Yes, this happened because I needed to rename the generic function called `getPluginEntryPoint` and create a function for the fields and rules.

Reply:
Np, I just got confused by the diff and thought we were just passing Rules... well git problems